lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -217,7 +205,9 @@ namespace magma {
 
 AsyncPipelinedClient::AsyncPipelinedClient(
     std::shared_ptr<grpc::Channel> channel)
-    : stub_(Pipelined::NewStub(channel)) {}
+    : stub_(Pipelined::NewStub(channel)) {
+  teid = 0;
```

Comment:
Don't start this TEID from zero as MME in 4G also starts with 0. Define something like M5G_MIN_TEID and set it equal to `UINT32_MAX/2`. In the future, we should randomize TEID allocation both in 4G and 5G.

Reply:
Defined, redefined.